README.md

```diff
@@ -36,6 +36,28 @@ python build-ansible-files.py monasca_config.yml
 ansible-playbook -i ./hosts site.yml
 ```
 
+extras
+======
+
+## Optional ansible alarm configuration
+
+One time setup:
+```
+cd monasca_installer
+git subtree add --prefix monasca_installer/library/monasca https://github.com/hpcloud-mon/ansible-module-monasca.git master --squash
+```
+
+Modify alarms.yml monasca_notification_method.address:
```

Comment:
The options for these default alarms could be made into variables so it is just part of the config and not a special modification of the playbook.